2.4/root/usr/share/container-scripts/mongodb/common.sh

```diff
@@ -86,8 +86,12 @@ function _wait_for_mongo() {
 # To get list of endpoints, you need to have headless Service named 'mongodb'.
 # NOTE: This won't work with standalone Docker container.
 function endpoints() {
-  service_name=${MONGODB_SERVICE_NAME:-mongodb}
-  dig ${service_name} A +search +short 2>/dev/null
+  if [ -n "${MONGODB_REPLICA_MEMBERS+x}" ]; then
```

Comment:
It's not only personal preference but also the style that is used in this repository:
```bash
$ git grep -c ':-}' | wc -l
16
$ git grep -c '+}' | wc -l
0
```
All of these cases are cryptic but it's better to use the same approach everywhere in the repository instead of different styles.